Shared/Samples/Query feature table/QueryFeatureTableView.swift

```diff
@@ -0,0 +1,135 @@
+                        .onSubmit {
+                            // Set the query string when the search button is tapped.
+                            query = model.currentQuery
+                        }
+                        .submitLabel(.search)
+                        Button("Cancel") {
+                            // Hide keyboard.
+                            searchFieldIsFocused = false
+                        }
+                    }
+                    .padding()
+                }
+                .task(id: query) {
+                    // Makes sure we have a query string and a layer.
+                    guard !query.isEmpty, let featureLayer = model.featureLayer else { return }
+                    // Unselect all selected features.
+                    featureLayer.unselectFeatures(model.selectedFeatures)
+                    model.selectedFeatures.removeAll()
+                    // Make the query parameters and execute the query.
+                    let queryParameters = QueryParameters()
+                    queryParameters.whereClause = "upper(STATE_NAME) LIKE '%\(query.uppercased())%'"
+                    do {
+                        let queryResult = try await model.featureTable.queryFeatures(using: queryParameters)
+                        model.selectedFeatures = Array(queryResult.features())
+                        if !model.selectedFeatures.isEmpty {
+                            // Display the selection.
+                            featureLayer.selectFeatures(model.selectedFeatures)
+                            // Zoom to selected feature.
+                            if let featureGeometry = model.selectedFeatures.first?.geometry {
+                                try await mapViewProxy.setViewpointGeometry(featureGeometry, padding: 25)
+                            }
+                        } else {
+                            // If the query returned no features then zoom
+                            // to the extent of the layer.
+                            if let layerExtent = featureLayer.fullExtent {
+                                try await mapViewProxy.setViewpointGeometry(layerExtent, padding: 50)
+                            }
+                        }
+                    } catch {
+                        // Display error as an alert.
+                        self.error = error
+                    }
+                }
+        }
+    }
+}
+
+private extension QueryFeatureTableView {
+    class Model: ObservableObject {
+        /// A map with a topographic basemap style.
+        let map = Map(basemapStyle: .arcGISTopographic)
+        
+        @Published var currentQuery = ""
+        
+        var selectedFeatures: [Feature] = []
+        
+        // Create a feature table using a url.
+        let featureTable = ServiceFeatureTable(
+             url: URL(string: "https://services.arcgis.com/jIL9msH9OI208GCb/arcgis/rest/services/USA_Daytime_Population_2016/FeatureServer/0")!
+)
```

Comment:
```suggestion
        )
```
The indentation was off in my first suggestion.